quickstack/manifests/sahara.pp

```diff
@@ -209,6 +209,13 @@
     after   => '        curr_extra = job_execution.extra.copy()',
   }
 
+  file_line { 'mr_job':
+    notify  => Service['httpd'],
+    path    => '/usr/share/openstack-dashboard/openstack_dashboard/contrib/sahara/content/data_processing/jobs/workflows/create.py',
+    line    => "                'data-jobtype-mapreduce': (\"Choose libraries\"),",
```

Comment:
I am not sure what the significance of leading _ is in "_(Choose libraries)" but it looks like all the other lines have a leading _. Maybe we should add it to this commit as well?

```
           attrs={
                'class': 'switched',
                'data-switch-on': 'jobtype',
                'data-jobtype-pig': _("Choose libraries"),
                'data-jobtype-hive': _("Choose libraries"),
                'data-jobtype-shell': _("Choose additional files"),
                'data-jobtype-spark': _("Choose libraries"),
                'data-jobtype-java': _("Choose libraries"),
                'data-jobtype-mapreduce': ("Choose libraries"),
                'data-jobtype-mapreduce.streaming': _("Choose libraries")
            }
```